src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -16,6 +21,7 @@ public class CookieRequestCultureProvider : RequestCultureProvider
         private static readonly char[] _cookieSeparator = new[] { '|' };
         private static readonly string _culturePrefix = "c=";
         private static readonly string _uiCulturePrefix = "uic=";
+        private static ILogger _logger;
```

Comment:
In this case we need to duplicate the code to get the `culture` and `ui-culture` like what we have in `ParseCookieValue()` to use them in logging